Common/Data/Consolidators/RenkoConsolidator.cs

```diff
@@ -68,6 +68,11 @@ public RenkoConsolidator(decimal barSize, RenkoType type)
                 throw new ArgumentException($"RenkoConsolidator can only be initialized with RenkoType.Wicked. For RenkoType.Classic, please use the other constructor overloads.");
             }
 
+            if (barSize < Extensions.GetDecimalEpsilon())
+            {
+                throw new ArgumentOutOfRangeException(nameof(barSize), "RenkoConsolidator bar size must be positve and greater than 1e-28");
```

Comment:
Cool! 
Could we have the constructors call each other? With these smart constructors, I think it's a nice approach to have them call each other to avoid having to duplicate this logic so many times, which is error-prone too and actually lead to this issue

Reply:
1e-28 is super small -- I'm guessing an out of range exception brought us here. In my own checks I'll usually limit it to 1e-20 to avoid divisions pushing me out of range.